cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -127,6 +134,11 @@ type CloudProvider interface {
 	// GetAvailableGPUTypes return all available GPU types cloud provider supports.
 	GetAvailableGPUTypes() map[string]struct{}
 
+	// GetNodeAccelerator returns the label and resource name for the accelerator added to node.
+	GetNodeAccelerator(*apiv1.Node) Accelerator
```

Comment:
nit: I think it's worth to consider for this method to return the Accelerator pointer rather than struct. That way we would be able to better indicate the "no accelerator present" behaviour.

Also, there should be some info in the comment above about what happens if the node has no accelerator.